pkg/pillar/cmd/zedmanager/memorysizemgmt.go

```diff
@@ -43,11 +43,13 @@ func getRemainingMemory(ctxPtr *zedmanagerContext) (uint64, uint64, uint64, erro
 	usedMemorySize += uint64(memoryReservedForEve)
 	deviceMemorySize, err := sysTotalMemory(ctxPtr)
 	if err != nil {
-		return 0, 0, 0, err
+		ctxPtr.checkFreedResources = true
+		return 0, 0, 0, fmt.Errorf("sysTotalMemory failed: %v. Schedule checkRetry", err)
 	}
 	if usedMemorySize > deviceMemorySize {
-		log.Errorf("getRemainingMemory discrepancy: accounted apps: %s; usedMemorySize: %d; deviceMemorySize: %d",
+		log.Errorf("getRemainingMemory discrepancy: accounted apps: %s; usedMemorySize: %d; deviceMemorySize: %d. Schedule checkRetry",
```

Comment:
Ditto

Reply:
This case is not a direct error. I mean, I do not assign it to app, only show in logs.

Reply:
Yeah but "schedule" is imperative (asking someone to do this), as opposed to "scheduling" which is what the code actually does.